bin/i18n/sync-out.rb

```diff
@@ -181,6 +181,8 @@ def restore_redacted_files
           plugins << 'vocabularyDefinition'
         elsif original_path.starts_with? "i18n/locales/original/curriculum_content"
           plugins.push(*Services::I18n::CurriculumSyncUtils::REDACT_RESTORE_PLUGINS)
+        elsif File.fnmatch('i18n/locales/original/blockly-mooc/*lab.json', original_path)
```

Comment:
No super sure about this `restore` pattern. But I wanted to capture all `*lab.json` files.

Reply:
Same as other comment.